data/mods/bio_mod/b_bio_weapons.json

```diff
@@ -0,0 +1,53 @@
+[  
+  {
+    "id": "bio_emp_gun",
+    "type": "GUN",
+    "name": "EMP Projector",
+    "description": "A ranged emp generator system is implanted on the palm of the user's right hand and arm.  Fires super-concentrated electric pulses for a short distance.  Extremely effective against electronic targets but mostly useless otherwise.",

```

Comment:
There should be consistency in descriptions. In `bio_scalpel` description you wrote "...on _your_ fingers...", here you write "...on the palm of the _user's_ right hand..."